bukkit/src/main/java/net/william278/husktowns/hook/PlaceholderAPIHook.java

```diff
@@ -54,6 +57,17 @@ public static class HuskTownsExpansion extends PlaceholderExpansion {
 
         private static final String NOT_IN_TOWN_COLOR = "#aaaaaa";
         private static final String WILDERNESS_COLOR = "#2e2e2e";
+        private static final BigDecimal THOUSAND = BigDecimal.valueOf(1000);
+        private static final String DECIMAL_FORMAT = "#.#";
+        private static final TreeMap<BigDecimal, String> NUMBER_FORMAT = new TreeMap<>();
+
+        static {
+            NUMBER_FORMAT.put(THOUSAND, "k");
+            NUMBER_FORMAT.put(THOUSAND.pow(2), "M");
+            NUMBER_FORMAT.put(THOUSAND.pow(3), "G");
+            NUMBER_FORMAT.put(THOUSAND.pow(4), "T");
+            NUMBER_FORMAT.put(THOUSAND.pow(5), "Q");
+        }
```

Comment:
Perhaps new `TreeMap<>(Map.of(...))` with the number formats instead of a static block here?